lib/lcformvalidation.d.ts

```diff
@@ -27,8 +25,24 @@ export interface FormValidationFunction {
   (vm: any): ValidationResult;
 }
 
+export interface FieldValidationFunction {
+  (value: any, vm: any): ValidationResult;
+}
+
+export interface FieldValidationConstraint extends Object {
+  validator: FieldValidationFunction;
+  trigger?: { [key: string]: boolean },
```

Comment:
You are declaring an interface `ValidationFilter` below. Do you have to type `trigger` with this interface?